### PR TITLE
Add wallet standard support

### PR DESCRIPTION
**Pull request by Perronef5**

### CODE REVIEW

File: components/App.tsx [hunk 1]
```diff
@@ -250,13 +249,15 @@ export function App(props: Props) {
         <QueryClientProvider client={queryClient}>
           <ThemeProvider defaultTheme="Dark">
             <WalletIdentityProvider appName={'Realms'}>
+              <WalletProvider>
```
Comment (nramadas) [inline components/App.tsx:252]: i think the next line(s) need to be indented

**Review by nramadas — CHANGES_REQUESTED** [review]
just a few small fixups

File: components/App.tsx (hunk 1 above)
Comment (Perronef5) [reply components/App.tsx:252]: Done!

File: hooks/useInitWallet.tsx [hunk 2]
```diff
@@ -51,10 +53,14 @@ export default function useInitWallet() {
     // eslint-disable-next-line react-hooks/exhaustive-deps -- TODO please fix, it can cause difficult bugs. You might wanna check out https://bobbyhadz.com/blog/react-hooks-exhaustive-deps for info. -@asktree
   }, [selectedProviderUrl, savedProviderUrl])
 
-  const provider = useMemo(() => getWalletProviderByUrl(selectedProviderUrl), [
+  const provider = useMemo(() => getWalletProviderByUrl(selectedProviderUrl, wallets), [
     selectedProviderUrl,
+    wallets,
   ])
 
+  useEffect(() => {
```
[the remark this answers is not in the document]
Comment (Perronef5) [reply hooks/useInitWallet.tsx]: Nope. Removed!

**Pull request merged**